challenges/cat-photo-gallery/script.js

```diff
@@ -0,0 +1,36 @@
+window.onload = getRandomCatPicture ()
+
+async function getRandomCatPicture () {
+    await fetch("https://cataas.com/cat?json=true")
```

Comment:
hi 
well done Denis, that you figured how to use `cataas.com` api
however, at this line you have one small issue. 
your `getRandomCatPicture()` function is declared as async. 
btw, a separate well done for the very good name of the function! :) 
so, how do you use fetch inside an async function with await? 
can you google that? you don't need `then(....)` in there.